deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore-cluster/base/dhstore/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dhstore
+spec:
+  template:
+    spec:
+      topologySpreadConstraints:
+        - maxSkew: 1
+          topologyKey: topology.kubernetes.io/zone
+          whenUnsatisfiable: ScheduleAnyway
+      containers:
+        - name: dhstore
+          args:
+            - '--storePath=/data'
+            - '--disableWAL'
```

Comment:
Do we want to keep WAL on? We disabled it originally to improve performance but that might not be the case for the cluster anymore.